### DISCUSSION TITLE
The recognition accuracy of Chinese microphone voice input is too low

### DISCUSSION THREAD

**Discussion by Arkzieo** [vivekuppal/transcribe#244] [Ideas]
The recognition accuracy of Chinese microphone voice input is too low, which is much worse than Microsoft's built-in voice recognition capability.I don't know if it's because there is no contextual input.

**Comment by vivekuppal** [comment]
It would be helpful to have an audio file with the transcribe provided transcription and MS provided transcription.

**Reply by Arkzieo** [reply]
This may be because I am using the base model,
local_transcripton_model_file: 'base'
 which is not as good as the current MS speech conversion service that requires an internet connection. Specifically, the Chinese recognition rate of the base model is very low, with an error rate of almost 50%.
local_transcripton_model_file: 'medium'
![image](https://github.com/user-attachments/assets/f373d83e-0f6b-4cf9-a249-fcd0c4f45eb5)
![图片1](https://github.com/user-attachments/assets/b4d097a1-f264-47e7-8a95-b8ee6b15bb54)
The recognition rate of medium is still not high. By the way, I got the download link directly from your docs/ModelSelection.md documentation, and then put the model file in the transcribe.exe/models/ directory. I don't know if this correctly enabled the model.
but I think this is normal, thank you for your reply.
